Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -114,6 +114,16 @@ export default class LinearTimelineComponent extends Vue {
         this.applyTimelineFilter();
     }
 
+    @Watch("visibleTimelineEntries")
+    private onVisibleEntriesUpdate() {
+        if (this.visibleTimelineEntries.length > 0) {
+            this.eventBus.$emit(
+                "timelinePageUpdate",
```

Comment:
same comment above for using const instead

Reply:
same as above